shared/fetch-hg-map.sh

```diff
@@ -14,9 +14,15 @@ if [ ! -f "$WORKING/git_hg.map" ]; then
     # results in a 503 error so we just get the map entries from some fixed recent date.
     date=2019-01-01
 
+    rm -f "$WORKING/git_hg.map.tmp"
```

Comment:
Not clear why this is needed if you delete it after merging them anyway

Reply:
Well, just in case there was garbage from a previous run... I don't think it hurts.